chore: updated screenshots in README.md and added recent events (#17650)
apache/superset: README.md

[**Resources**](#resources) |
[**Organizations Using Superset**](RESOURCES/INTHEWILD.md)

## Why Superset?

Superset is a modern data exploration and data visualization platform. Superset can replace or augment proprietary business intelligence tools for many teams.

Superset provides:

- A **no-code interface** for building charts quickly

> - A powerful, web-based **SQL Editro** for advanced querying

Reviewer: ```suggestion
- A powerful, web-based **SQL Editor** for advanced querying
```

Reviewer: Actually, not sure if we should capitalize Editor here.

Reviewer 2: I vote for capitalize! It's a feature name

- A **lightweight semantic layer** for quickly defining custom dimensions and metrics
- Out of the box support for **nearly any SQL** database or data engine
- A wide array of **beautiful visualizations** to showcase your data

> metadata browser

Reviewer: This line looks wrong... is this a separate item?

- Lightweight, configurable **caching layer** to help ease database load
- Highly extensible **security roles and authentication** options
- An **API** for programmatic customization
- A **cloud-native architecture** designed from the ground up for scale

## Screenshots & Gifs

**Large Gallery of Visualizations**

<kbd><a href="https://superset.apache.org/gallery"><img title="Gallery" src="superset-frontend/src/assets/images/screenshots/gallery.jpg"/></a></kbd><br/>

**Craft Beautiful, Dynamic Dashboards**

<kbd><img title="View Dashboards" src="superset-frontend/src/assets/images/screenshots/slack_dash.jpg"/></kbd><br/>

> **No Code Chart Builder**

Reviewer: ```suggestion
**No-Code Chart Builder**
```
This seems like it should be hyphenated. Maybe. ¯\\\_(ツ)_/¯

**Slice & dice your data**
**No Code Chart Builder**

<kbd><img title="Slice & dice your data" src="superset-frontend/src/assets/images/screenshots/explore.jpg"/></kbd><br/>

**Query and visualize your data with SQL Lab**
**Powerful SQL Editor**

<kbd><img title="SQL Lab" src="superset-frontend/src/assets/images/screenshots/sql_lab.jpg"/></kbd><br/>

> **Visualize geospatial data with deck.gl**

Reviewer: I'm OK with dropping the section, but for SEO, it seems like we might want to keep the word "geospatial" in the page somewere.

> Superset can query data from any SQL-speaking datastore or data engine (e.g. Presto, Trino, or Athena) that has a Python DB-API driver and a SQLAlchemy dialect.

Reviewer: Should we just make a bullet list of off-the-shelf connectors, for SEO purposes ad/or general completionisim?

Author: We have a list here (https://superset.apache.org/docs/databases/installing-database-drivers/). My general feeling is I want to actually reduce the amount of content in the readme and move more of it to the docs / apache site. This seems consistent anyway with most other open source projects have done (e.g. kafka, spark). As a counter point tho, Airflow still has a bunch of content on their readme.

Maybe for now we just do logos, in my next pass I can enumerate the full list?

Reviewer: ```suggestion
Superset can query data from any SQL-speaking datastore or data engine (Presto, Trino, Athena, [and more](https://superset.apache.org/docs/databases/installing-database-drivers/)) that has a Python DB-API driver and a SQLAlchemy dialect.
```
Let's use the link, then!
